### DISCUSSION TITLE
How to setup a standalone project without Symfony?

### DISCUSSION THREAD

**Discussion by sebastianstucke87** [api-platform/core#5640] [Q&A] [answered]
Hello,

for educational purposes, I would like to setup a standalone project without Symfony. According to the documentation:
- ["API Platform is also designed as a set of independent and reusable components. (...)"](https://api-platform.com/)
- ["It can be used as a standalone or with the Symfony framework (...)"](https://api-platform.com/docs/core/#the-api-platform-core-library)

For example, the Symfony router can be easily used without the Symfony framework: https://github.com/symfony/routing#getting-started

Now I'm wondering on how to do this with **"api-platform/core"**, since all documentation examples seem to be for a Symfony project with Symfony (yaml-) configuration. Did I missunderstand the meaning of "standalone"?

**Comment by dunglas** [comment, marked as answer]
You can find the documentation to use API Platform without Symfony here: https://api-platform.com/docs/core/bootstrap/

**Reply by sebastianstucke87** [reply]
Thanks!
